exonum-java-binding/testkit/src/test/java/com/exonum/binding/testkit/TestKitTest.java

```diff
@@ -128,55 +237,209 @@ void createTestKitWithoutServices() {
+        checkTransactionsCommittedSuccessfully(view, block, message, message2);
+        return null;
+      });
     }
+  }
 
-    @Override
-    public String getName() {
-      return SERVICE_NAME;
+  @Test
+  void createBlockWithTransactionsVarargs() {
+    try (TestKit testKit = TestKit.forService(TestServiceModule.class)) {
+      TransactionMessage message = constructTestTransactionMessage("Test message");
+      TransactionMessage message2 = constructTestTransactionMessage("Test message 2");
+
+      Block block = testKit.createBlockWithTransactions(message, message2);
+      assertThat(block.getNumTransactions()).isEqualTo(2);
+
+      testKit.withSnapshot((view) -> {
+        checkTransactionsCommittedSuccessfully(view, block, message, message2);
+        return null;
+      });
     }
+  }
 
-    @Override
-    public Transaction convertToTransaction(RawTransaction rawTransaction) {
-      throw new UnsupportedOperationException();
+  private TransactionMessage constructTestTransactionMessage(String payload) {
+    return TransactionMessage.builder()
+        .serviceId(TestService.SERVICE_ID)
+        .transactionId(TestTransaction.ID)
+        .payload(payload.getBytes(BODY_CHARSET))
+        .sign(KEY_PAIR, CRYPTO_FUNCTION);
+  }
+
+  private void checkTransactionsCommittedSuccessfully(
+      View view, Block block, TransactionMessage message, TransactionMessage message2) {
+    Blockchain blockchain = Blockchain.newInstance(view);
+    assertThat(blockchain.getHeight()).isEqualTo(1);
+    assertThat(block).isEqualTo(blockchain.getBlock(1));
+    Map<HashCode, TransactionResult> transactionResults = toMap(blockchain.getTxResults());
+    assertThat(transactionResults).hasSize(2);
+
+    TransactionResult transactionResult = transactionResults.get(message.hash());
+    assertThat(transactionResult).isEqualTo(TransactionResult.successful());
+    TransactionResult transactionResult2 = transactionResults.get(message2.hash());
+    assertThat(transactionResult2).isEqualTo(TransactionResult.successful());
+  }
+
+  @Test
+  void createBlockWithTransactionWithWrongServiceId() {
+    try (TestKit testKit = TestKit.forService(TestServiceModule.class)) {
+      short wrongServiceId = (short) (TestService.SERVICE_ID + 1);
+      TransactionMessage message = TransactionMessage.builder()
+          .serviceId(wrongServiceId)
+          .transactionId(TestTransaction.ID)
+          .payload("Test message".getBytes(BODY_CHARSET))
+          .sign(KEY_PAIR, CRYPTO_FUNCTION);
+      IllegalArgumentException thrownException = assertThrows(IllegalArgumentException.class,
+          () -> testKit.createBlockWithTransactions(message));
+      RawTransaction rawTransaction = TestKit.toRawTransaction(message);
+      String expectedMessage = String.format("Unknown service id (%s) in transaction (%s)",
+          wrongServiceId, rawTransaction);
+      assertTrue(thrownException.getMessage().contains(expectedMessage));
```

Comment:
Here and below: assertTrue shall not be used where matchers are available because you won't get expected/actual in case of failure in the output (assertThat ... contains|matches|equals)